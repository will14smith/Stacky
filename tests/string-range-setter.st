main () -> () { "abcdef" "XXX" 1 !!~ print }