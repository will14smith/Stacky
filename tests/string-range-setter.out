=== Output ===
aXXXef

=== Exit ===
0
